packages/fx-core/templates/plugins/resource/aad/auth/V3/README.txt

```diff
@@ -1 +1,663 @@
+        with:
+          manifestTemplatePath: "YOUR_PATH_TO_AAD_APP_MANIFEST" # Relative path to teamsfx folder. Environment variables in manifest will be replaced before apply to AAD app
+          outputFilePath : ./build/aad.manifest.${{TEAMSFX_ENV}}.json
+      # Output: following environment variable will be persisted in current environment's .env file.
+      # AAD_APP_ACCESS_AS_USER_PERMISSION_ID: the id of access_as_user permission which is used to enable SSO
+      ```
+      Note: Replace the value of "manifestTemplatePath" with the relative path of AAD app manifest noted in step 1.
+            For example, './aad.manifest.template.json'
+
+  In `teamsfx/app.local.yml` only:
+    - Add following lines under `configureApp` to add AAD related configs to local debug service.
+      ```
+      - uses: appsettings/generate
+        with:
+          target: ./appsettings.Development.json
+          appsettings:
+            TeamsFx:
+              Authentication:
+                ClientId: ${{AAD_APP_CLIENT_ID}}
+                ClientSecret: ${{SECRET_AAD_APP_CLIENT_SECRET}}
+                OAuthAuthority: ${{AAD_APP_OAUTH_AUTHORITY}}
+      ```
+  -------------------------
+
+4. Update Infra
+  AAD related configs needs to be configure to your remote service. Following example shows the configs on Azure Webapp.
+    a. TeamsFx__Authentication__ClientId: AAD app client id
+    b. TeamsFx__Authentication__ClientSecret: AAD app client secret
+    c. TeamsFx__Authentication__OAuthAuthority: AAD app oauth authority
+  
+  -------------------------
+  Example for TeamsFx Tab template
+  -------------------------
+  Open `infra/azure.parameter.json` and add following lines into `parameters`:
+  ```
+  "tabAadAppClientId": {
+    "value": "${{AAD_APP_CLIENT_ID}}"
+  },
+  "tabAadAppClientSecret": {
+    "value": "${{SECRET_AAD_APP_CLIENT_SECRET}}"
+  },
+  "tabAadAppOauthAuthorityHost": {
+    "value": "${{AAD_APP_OAUTH_AUTHORITY_HOST}}"
+  },
+  "tabAadAppTenantId": {
+    "value": "${{AAD_APP_TENANT_ID}}"
+  }
+  ```
+
+  Open `infra/azure.bicep` find follow line:
+  ```
+  param location string = resourceGroup().location
+  ```
+  and add following lines:
+  ```
+  param tabAadAppClientId string
+  param tabAadAppOauthAuthorityHost string
+  param tabAadAppTenantId string
+  @secure()
+  param tabAadAppClientSecret string
+  ```
+  and add following configs in webApp.properties.siteConfig.appSettings
+  ```
+  {
+    name: 'TeamsFx__Authentication__ClientId'
+    value: tabAadAppClientId
+  }
+  {
+    name: 'TeamsFx__Authentication__ClientSecret'
+    value: tabAadAppClientSecret
+  }
+  {
+    name: 'TeamsFx__Authentication__OAuthAuthority'
+    value: uri(tabAadAppOauthAuthorityHost, tabAadAppTenantId)
+  }
+  ```
+  -------------------------
+
+5. Update `appsettings.json` and `appsettings.Development.json`
```

Comment:
Since remote gets appsettings from web app configuration, local uses appsettings/generate to update appsettings.json, do we still need to ask users modify appsettings.json here?

Reply:
Add todo and I will confirm with zhaofeng